apps/namadillo/src/App/Common/Navigation.tsx

```diff
@@ -69,14 +69,20 @@ export const Navigation = (): JSX.Element => {
             </a>
           </li>
         </ul>
-        <ul className="flex items-center gap-4 text-yellow text-2xl transition-colors duration-300 ease-out-quad">
+        <ul className="flex items-center gap-4 text-yellow text-2xl">
           <li>
-            <a href={TWITTER_URL} className="hover:text-cyan">
+            <a
+              href={TWITTER_URL}
+              className="transition-colors duration-150 hover:text-cyan"
```

Comment:
I think my eyes are getting old because I just cannot see this animation at 150ms :older_man:

Reply:
I've increased the value a bit to me more inclusive ~I couldn't see that as well~